### ISSUE TITLE
Facebook sharing - has no photo showing up

### ISSUE THREAD

**Issue by futuresoup** [mapseed/platform#70]

**Comment by LukeSwart** [comment]
Can you elaborate on this a little more? I'm not sure what you mean.

**Comment by futuresoup** [comment]
When you paste a link into Facebook to a raingarden, Facebook doesn't properly scrape the page and display a photo. You can use their [Debug Tool](http://developers.facebook.com/tools/debug) to check on your page being meta tagged properly for their scraper

**Comment by LukeSwart** [comment]
This was partially fixed on PR #220, so our homepage and default links have a decent thumbnail showing up. But we also want a thumbnail of a place's photo attachment to show on Facebook when someone shares the place. Right now, it just defaults to the homepage's thumbnail (which has just been fixed).

We have to explore this a little more, but I think it has to do with the following code under the flavor's `templates/index.html` file:

```
    {% with attachment=place.properties.attachments|first %}
    <meta name="og:image" content="{{ attachment.file }}">
    {% endwith %}
```

**Comment by futuresoup** [comment]
Duplicate of #292
